Maddori.Apple/Maddori.Apple/Screen/Setup/SetNickname/SetNicknameViewController.swift

```diff
@@ -197,13 +198,22 @@ final class SetNicknameViewController: BaseViewController {
     // MARK: - func
     
     private func didTappedProfile() {
-        // FIXME: - 갤러리로 이동
-        print("프로필 누름")
+        let actionSheet = UIAlertController(title: TextLiteral.setNicknameControllerProfileActionSheetTitle, message: nil, preferredStyle: .actionSheet)
+        let libraryAction = UIAlertAction(title: TextLiteral.setNicknameControllerProfileActionSheetLibraryTitle, style: .default) { _ in self.openLibrary() }
+        let cameraAction = UIAlertAction(title: TextLiteral.setNicknameControllerProfileActionSheetCameraTitle, style: .default) { _ in self.openCamera() }
+        let cancelAction = UIAlertAction(title: TextLiteral.setNicknameControllerProfileActionSheetCancelTitle, style: .cancel)
+        
+        actionSheet.addAction(libraryAction)
+        actionSheet.addAction(cameraAction)
+        actionSheet.addAction(cancelAction)
+        present(actionSheet, animated: true, completion: nil)
```

Comment:
p3;

```suggestion
        makeActionSheet(title: TextLiteral.setNicknameControllerProfileActionSheetTitle,
            actionTitles: [TextLiteral.setNicknameControllerProfileActionSheetLibraryTitle,
                                       TextLiteral.setNicknameControllerProfileActionSheetCameraTitle,
                                       TextLiteral.setNicknameControllerProfileActionSheetCancelTitle],
                        actionStyle: [.default, .default, .cancel],
                        actions: [ { _ in self.openLibrary()}, { _ in self.openCamera()}, nil])
```

저희는 `makeAlert`이랑, `makeActionSheet`를 따로 만들어 사용하고 있기 때문에 위와 같이 바꿔도 괜찮을 것 같아요 !

Reply:
<img width="815" alt="스크린샷 2023-02-02 오전 11 25 48" src="https://user-images.githubusercontent.com/59243274/216215985-089e2762-7c53-4c2a-92e3-a9fa7c73b09a.png">

요게 있답니다!

Reply:
[9f2ebe0](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/291/commits/9f2ebe03d60c2d3b1ed98c541c0dd082af12885b)
와우 ㅋㅋㅋ 감사해요! 덕분에 코드가 많이 줄었어요! 쵝오